srcpkgs/busybox/template

```diff
@@ -1,11 +1,11 @@
 # Template file for 'busybox'
 pkgname=busybox
 version=1.34.1
-revision=1
+revision=2
 hostmakedepends="perl"
 checkdepends="tar which zip"
 short_desc="Swiss Army Knife of Embedded Linux"
-maintainer="Cameron Nemo <cnemo@tutanota.com>"
+maintainer="Lucas Roberto <sigmw@protonmail.com>"
```

Comment:
Lol have fun :+1:

Reply:
Thought this should be changed for the next update, no? If not, sorry.

Reply:
Oh generally the maintainer is not removed unless they are inactive or purposefully moving maintainership.

Reply:
If you want i can keep my name  👀 👀 👀

Reply:
Just kidding! I reverted. (👀 )